climada/entity/exposures/litpop/nightlight.py

```diff
@@ -58,24 +59,26 @@
 NASA_SITE = 'https://www.nasa.gov/specials/blackmarble/*/tiles/georeferrenced/'
 """NASA nightlight web url."""
 
-BM_FILENAMES = ['BlackMarble_*_A1_geo_gray.tif',
-                'BlackMarble_*_A2_geo_gray.tif',
-                'BlackMarble_*_B1_geo_gray.tif',
-                'BlackMarble_*_B2_geo_gray.tif',
-                'BlackMarble_*_C1_geo_gray.tif',
-                'BlackMarble_*_C2_geo_gray.tif',
-                'BlackMarble_*_D1_geo_gray.tif',
-                'BlackMarble_*_D2_geo_gray.tif'
+BM_FILENAMES = ['BlackMarble_%i_A1_geo_gray.tif',
+                'BlackMarble_%i_A2_geo_gray.tif',
+                'BlackMarble_%i_B1_geo_gray.tif',
+                'BlackMarble_%i_B2_geo_gray.tif',
+                'BlackMarble_%i_C1_geo_gray.tif',
+                'BlackMarble_%i_C2_geo_gray.tif',
+                'BlackMarble_%i_D1_geo_gray.tif',
+                'BlackMarble_%i_D2_geo_gray.tif'
                ]
 """Nightlight NASA files which generate the whole earth when put together."""
 
-def check_required_nl_files(bbox, *coords):
+BM_YEARS = [2016, 2012] # list of available years with data, please update.
```

Comment:
This should then be moved to the configuration file where it is easier to maintain. Maybe even a bot can be written that checks for the released data regularly?